test/fuzz/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if (${SPIRV_BUILD_FUZZER})
+  add_spvtools_unittest(TARGET fuzz
+        SRCS
+        fuzz_test_util.h
```

Comment:
Maybe pull the source files out into a
```
set(SOURCES
   fuzz_test_util.h
   fuzz_test_util.cpp
   transformation_split_block_test.cpp)
```

Then reference that here.

Reply:
Done.